Gems/PhysX/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -845,11 +854,96 @@ namespace PhysX
         }
         return AZ::Vector3::CreateZero();
     }
+
+    const AzPhysics::SimulatedBody* ArticulationLinkComponent::GetSimulatedBodyConst() const
+    {
+        const AZ::Entity* rootEntity = GetArticulationRootEntity();
+        const auto rootComponent = rootEntity->FindComponent<ArticulationLinkComponent>();
+
+        return AZ::Interface<AzPhysics::SceneInterface>::Get()->GetSimulatedBodyFromHandle(
+            rootComponent->m_attachedSceneHandle, GetSimulatedBodyHandle());
```

Comment:
There should be either `nullptr` checks or `AZ_Asserts` to protect against null pointer exceptions for `rootEntity` and `rootComponent` here.

Reply:
Agree!